content/en/users/tutorials/htc-job-submission/_index.md

```diff
@@ -0,0 +1,589 @@
+---
+title: "Submitting HTC Jobs"
+type: docs
+weight: 160
+description: >
+  Submitting High Throughput Compute jobs
+---
+
+## Overview
+
+This tutorial describes how to submit
+[High Throughput Compute (HTC)](../../compute/high-throughput-compute) jobs
+using command-line.
+
+## Prerequisites
+
+To submit an EGI HTC job, you will have to:
+
+1. Obtain an X.509 user certificate. The supported certificates are issued by
+   Certification Authorities (CAs) part of the
+   [European Policy Management Authority for Grid Authentication (EUGridPMA)](https://www.eugridpma.org),
+   which is also part of the
+   [International Global Trust Federation (IGTF)](https://www.igtf.net).
+1. Enrol into a VO having access to HTC resources.
+
+> This tutorial will be using `dteam` a test Virtual Organisation that can be
+> used by resource providers, commands should be adjusted to the appropriate VO.
+
+## Step 1: getting access to a User Interface (UI)
+
+In order to interact with HTC resources, you should have access to a
+`User Interface`, often referred to as a `UI`. This software environment will
+provide all the tools required to interact with the different middleware, as
+different sites can be using different Computing Element (CE), such as
+HTCondorCE and ARC-CE (CREAM is a legacy software stack that is not officially
+supported).
+
+Different possibilities are available to access an UI:
+
+- Having access to an UI provided by/for your community
```

Comment:
Could we have one or two examples?

Reply:
There isn't any real generic examples, but some advanced community have or can provide some UI to their users, like by granting them access to a VM with all the required tool installed and the their specific VO configured. for CMS there is cmsconnect, but I'm not sure it would be helpful to mention this.